### DISCUSSION TITLE
Project maintainers

### DISCUSSION THREAD

**Discussion by petli** [python-xlib/python-xlib#273] [General]
#242 raised the question about the need for active project maintainers, so for better visibility I move it to a discussion. 

I can't claim to be a maintainer of this library since I haven't done anything actively for a number of years, because I don't use XWindows or even have any machine with it running any longer. I think @vasily-v-ryabov, who has been most active in getting new releases out, have said something similar. 

I think it would be much preferable to keep development within this repository, instead of it being forked of elsewhere and have confusion about what is the currently maintained version of the library. I still count as an admin here and could hand over to active users if there are no-one else around to do it, but if any other @python-xlib/admins sees this please chime in.

**Comment by ehfd** [comment]
I nominate @fransklaver to step up as maintainer, and also @Avasam. They have made substantial contributions. I still think @petli should hold that administrator post for emergency intervention in unpredictable situations.

We currently use Python-Xlib for https://github.com/selkies-project/selkies-gstreamer. I nominate prior contributor @danisla and (1 year later) myself. I have no capacity right now, but things will clear up for me in August-September 2025.

**Reply by fransklaver** [reply]
I wouldn't yet go as far as to say my contributions have been substantial, but thank you for the nomination and implied trust there.

In any case, a couple of active maintainers wouldn't hurt.

**Reply by ehfd** [reply]
I won't be accessing GitHub regularly from September until late 2025, so I'll see if @danisla might work.

**Reply by ehfd** [reply]
@petli I'd say make someone a maintainer first, then we could see who could be additional maintainers.

**Reply by Avasam** [reply]
I appreciate, but I'm not really familiar with xlib past interfacing with this library. And my contributions would stop at merging typeshed stubs into it to have first-party type annotations, with some CI checks to prevent regressions. (and fixing any bug that I come across that personally affects me)
